packages/tizen_rpc_port/lib/src/parcel.dart

```diff
@@ -0,0 +1,406 @@
+import 'dart:ffi';
+import 'dart:typed_data';
+import 'package:ffi/ffi.dart';
+
+import 'package:tizen_bundle/tizen_bundle.dart';
+import 'package:tizen_interop/6.5/tizen.dart';
+
+class _Timespec extends Struct {
+  @Int64()
+  external int tvSec;
+
+  @Int64()
+  external int tvNsec;
+}
+
+/// The timestamp when parcel created.
+class Timestamp {
+  Timestamp._(this.second, this.nanoSecond);
+
+  /// The second of timestamp.
+  int second;
+
+  /// THe nano second of timestamp. (10^-9 second)
+  int nanoSecond;
+}
+
+/// The header of parcel that is included information of the parcel.
+class ParcelHeader {
+  ParcelHeader._fromHandle(this._header);
+
+  late final Pointer<Void> _header;
+
+  /// The tag is used version naming from TIDL.
+
+  /// Gets the tag of header.
+  String get tag {
+    return using((Arena arena) {
+      final Pointer<Pointer<Char>> rawTag = arena();
+      final int ret = tizen.rpc_port_parcel_header_get_tag(_header, rawTag);
+      arena.using(rawTag.value, malloc.free);
+      if (ret != 0) {
+        throw ret;
+      }
+
+      return rawTag.value.toDartString();
+    });
+  }
+
+  /// Sets the tag of header.
+  set tag(String tag_) {
+    using((Arena arena) {
+      final Pointer<Char> rawTag = tag_.toNativeChar(allocator: arena);
+      final int ret = tizen.rpc_port_parcel_header_set_tag(_header, rawTag);
+      if (ret != 0) {
+        throw ret;
+      }
+    });
+  }
+
+  /// The sequence number is used when distinguish parcel pair from TIDL.
+
+  /// Gets the sequence number of header.
+  int get sequenceNumber {
```

Comment:
https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary

Reply:
I removed above descriptions.

Reply:
Also you don't have to write docs for both getter and setter. You can remove the doc from the setter and change the getter description as follows.

```suggestion
  /// The sequence number of this header.
  ///
  /// The sequence number is used by TIDL to identify each parcel.
  int get sequenceNumber {
```

Reply:
I removed setter description.
When we discussed, we removed the explanation for TIDL because I thought it would be better to remove it.